azure-pipelines.yml

```diff
@@ -229,4 +230,4 @@ steps:
     displayName: 'Upload staging zip to build artifacts'
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)/PTVSBuildArchive.zip'
-      ArtifactName: 'PTVS Staging Output'
```

Comment:
no newline, might break the yml

Reply:
Hmm weird. I actually used github to edit it. It added this new line.